src/main/java/com/glencoesoftware/bioformats2raw/Converter.java

```diff
@@ -975,6 +993,64 @@ private void processTile(
     writeBytes(zarr, shape, offset, tileBuffer);
   }
 
+  private void processChunk(
+      int series, int resolution, int plane, int xx, int yy, int zz,
+      int width, int height, int depth)
```

Comment:
Does it make sense to take ```int[] offset, int[] shape``` instead of the six position/length parameters?  That would be easier to extend to C/T chunks, and would eliminate the need for the checkstyle configuration change.